setup.py

```diff
@@ -24,23 +24,18 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
     ],
 
     python_requires='>=3.7',
 
     install_requires=[
         'affine',
         'boto3',
-        'fiona',
-        'gdal',
         'geopandas',
         'importlib_metadata',  # drop when py>=3.8
         'numpy',
-        'pyproj',
```

Comment:
You're calling pyproj in `rio_window.py`, so you should keep this one in `setup.py`